src/app/TideData.cpp

```diff
@@ -2,4 +2,39 @@
 // Created by swagner on 10/4/20.
 //
 
+#include <iostream>
+#include <include/nlohmann/json.hpp>
 #include "TideData.h"
+#include "TimeSeriesDataPoint.h"
+
+using json = nlohmann::json;
+
+// Time-series JSON; t is timestamp, v is value in feet
+TideData TideData::Parse(const std::string& data) {
+    json j = json::parse(data);
+
+    std::vector<TimeSeriesDataPoint> timeSeriesDataPoints;
+
+    for (auto & it : j) {
+        for(json::iterator it2 = it.begin(); it2 != it.end(); ++it2) {
+            auto entry = it2.value();
+            std::string timestamp;
+            std::string value;
+
+            entry["t"].get_to(timestamp);
+            entry["v"].get_to(value);
+
+            timeSeriesDataPoints.emplace_back(entry["t"], std::stof(value, nullptr));
```

Comment:
Should this be using timestamp
```suggestion
            timeSeriesDataPoints.emplace_back(timestamp, std::stof(value, nullptr));
```

Reply:
Why yes it should

Reply:
fixed